content/05.Experimental_Design.md

```diff
@@ -2,14 +2,19 @@
 
 <!-- Leader: Will Lewis; co-leaders: James Martin -->
 
-## Repetition 
+## Randomization 
+This section will discuss the importance of randomly assigning treatments to groups in experimental studies. The treatment is not ranomly applied to groups in observational studies, though methods such a random selection of study sites and survey areas can help to reduce bias. Some of this may overlap with previous sections describing the difference between experimental and observational studies.
 
 ## Replication 
+This section will discuss the importance of indepndent replicates for experimental and observation studies, including examples to show how incorrect inferences may be generated with insufficient levels of replication. Observational studies are correlative by nature, so inferences are generally weak without sufficient levels of replication. How to determine the experimental unit of analysis and avoiding the pitfalls of pseudoreplication will be discussed. This section will also discuss the value of a pilot study and performing a prospective power analysis for estimating the necessary level of replication within groups to observe a desired effect size.
```

Comment:
```suggestion
This section will discuss on the importance of independent replicates for experimental and observation studies, including examples to show how incorrect inferences may be generated with insufficient levels of replication. Observational studies are correlative by nature, so inferences are generally weak without sufficient levels of replication. How to determine the experimental unit of analysis and avoiding the pitfalls of pseudo-replication will be discussed. This section will also discuss the value of a pilot study and performing a prospective power analysis for estimating the necessary level of replication within groups to observe a desired effect size.
```

Reply:
Definitely need to coordinate with @ErlendNilsen on causation from observational studies. I think he will make the cause that causal inference is possible with observational studies if done in an appropriate way.